mypy/stubtest.py

```diff
@@ -1247,6 +1247,8 @@ def verify_typealias(
         "__author__",
         "__version__",
         "__copyright__",
+        # Created by `warnings.warn`, does not make much sense to have in stubs:
+        "__warningregistry__",
```

Comment:
I think this entry should go above the TODO comment on line 1246; it's here to stay :)